tests/config/bug_1758.cfg

```diff
@@ -0,0 +1,2043 @@
+#
```

Comment:
That needs a run through the reduction script.

I remember that you had a reason to postpone the usage of it but I don't quiet remember anymore why and if it is applicable for this case. I believe it was something related to the Unity fork.